app/src/main/java/org/cru/godtools/ui/dashboard/tools/ToolsFragment.kt

```diff
@@ -16,4 +17,10 @@ class ToolsFragment : BasePlatformFragment<DashboardToolsFragmentBinding>(R.layo
         binding.refresh.setupSwipeRefresh()
         binding.hasSpotlight = dataModel.hasSpotlight
     }
+
+    override fun onSyncData(helper: SwipeRefreshSyncHelper, force: Boolean) {
+        super.onSyncData(helper, force)
+        helper.sync(syncService.syncTools(force))
+        helper.sync(syncService.syncLanguages(force))
```

Comment:
why are you syncing languages from the `ToolsFragment`?

Reply:
I was thinking it was needed to sync the parallel and primary languages in sync.